site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -218,6 +218,8 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.allocator.generateTLS`                      | Set to true to generate TLS certificates or false to provide certificates in `certs/allocator/*`| `true`                 |
 | `agones.allocator.tolerations`                      | Allocator [toleration][toleration] labels for pod assignment                                    | `[]`                   |
 | `agones.allocator.affinity`                         | Allocator [affinity][affinity] settings for pod assignment                                      | `{}`                   |
+| `agones.allocator.disableTLS`                       | Turns off the built-in encryption and authentication mechanism for the agones-allocator service. | `false`                |
+| `agones.allocator.disableMTLS`                      | Just turns off client cert authentication for incoming connections to the allocator.            | `false`                |
```

Comment:
Could be simplified a bit, I cannot understand the difference based on the descriptions.
```suggestion
| `agones.allocator.disableMTLS`                      | Turns off client cert authentication for incoming connections to the allocator.            | `false`                |
```

Reply:
I clarified the text a bit.
disableMTLS - no client cert authentication
disableTLS -. full plaintext (no client authentication, and no server cert encryption)